app/wasm_config.go

```diff
@@ -0,0 +1,25 @@
+package app
+
+import (
+	wasmkeeper "github.com/CosmWasm/wasmd/x/wasm/keeper"
+)
+
+const (
+	// DefaultJunoInstanceCost is initially set the same as in wasmd
+	DefaultJunoInstanceCost uint64 = 60_000
+	// DefaultJunoCompileCost set to a large number for testing
```

Comment:
Is this comment still accurate?

Reply:
Not for testing, it's just set to a large number initially until we feel good about everything.